src/main/scala/com/mwz/sonar/scala/pr/github/Github.scala

```diff
@@ -40,24 +40,25 @@ object Github {
   def apply[F[_]: Sync](client: Client[F], pr: GlobalConfig.PullRequest): Github[F] =
     new Github[F] {
       val auth: Header = Header("Authorization", s"token ${pr.github.oauth}")
+      val userUri: String = s"${pr.github.apiurl}/user"
```

Comment:
Once you have an `Uri` here, you'll need to use the uri syntax, e.g. `pr.github.apiurl / "user"`.

Reply:
I've modified it. Please make sure that it's the right one.